### PR TITLE
Moving aria hook imports into first doc example

### PR DESCRIPTION
**Pull request by LFDanLu**
Closes https://github.com/adobe/react-spectrum/issues/2665

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

## 🧢 Your Project:

RSP

### CODE REVIEW

**Comment by LFDanLu** [comment]
haven't tested the docs yet, will do later

**Comment by LFDanLu** [comment]
Finished testing, looks fine to me locally. Note: merge in after https://github.com/adobe/react-spectrum/pull/2634

File: packages/@react-aria/ssr/docs/SSRProvider.mdx [hunk 1]
```diff
@@ -14,10 +14,6 @@ import docs from 'docs:@react-aria/ssr';
 import {HeaderInfo, PropTable} from '@react-spectrum/docs';
 import packageData from '@react-aria/ssr/package.json';
 
-```tsx import
-import {useCollator} from '@react-aria/ssr';
```
Comment (ktabors) [inline packages/@react-aria/ssr/docs/SSRProvider.mdx:18]: good catch that this wasn't used.

**Review by ktabors — APPROVED** [review]

**Comment by ktabors** [comment]
Did you run docs-differ, something else or is manual inspect the way to make sure this works?

**Review by snowystinger — DISMISSED** [review]
Went through all docs, everything appears to be in working order

**Comment by dannify** [comment]
Merge this after parcel

**Review by dannify — DISMISSED** [review]

**Pull request merged**